ledger/metrics/src/main/scala/com/daml/metrics/CacheMetrics.scala

```diff
@@ -18,6 +18,9 @@ final class CacheMetrics(override val prefix: MetricName, override val registry:
                     |incremented.""",
     qualification = Debug,
   )
+  @MetricDoc.GroupTag(
+    representative = "daml.execution.cache.<state_cache>.hits"
```

Comment:
Would it be possible to declare the `@MetricDoc.GroupTag(` at the place where it is used rather than where it is declared? I.e.  somewhere in the `ExecutionMetrics.scala` 
```
    val keyState: CacheMetrics = new CacheMetrics(prefix :+ "key_state", registry)
    val contractState: CacheMetrics = new CacheMetrics(prefix :+ "contract_state", registry)
```

Same comment applies to all similar metrics